wiki/Lua-Hyperspace-Module.md

```diff
@@ -1146,4 +1146,19 @@ local _, canMove = crew.extend:CalculateStat(Hyperspace.CrewStat.CAN_MOVE)
 - `std::vector<std::string>` `.hitShipSounds`
 - `std::vector<std::string>` `.hitShieldSounds`
 - `std::vector<std::string>` `.missSounds`
-- `string` `.image`
\ No newline at end of file
+- `string` `.image`
+
+## CombatControl
+
+### Fields
+- [`WeaponControl`](#weaponcontrol) `weapControl`
+- `Point` `.position`
+- `Point` `.targetPosition`
+- `bool` `.bossVisual`
```

Comment:
Variable is named `boss_visual`, not `bossVisual`. Also needs to be marked as immutable.